libcaf_net/caf/net/http/v1.cpp

```diff
@@ -32,8 +36,8 @@ writer& operator<<(writer& out, const std::string& str) {
 
 } // namespace
 
-std::pair<std::string_view, byte_span> split_header(byte_span bytes) {
-  std::array<std::byte, 4> end_of_header{{
+std::pair<std::string_view, byte_span> split_header(const byte_span bytes) {
+  static constexpr std::array<std::byte, 4> end_of_header{{
```

Comment:
Out of curiosity, what was your reasoning to make this static? Having these four bytes on the stack should pretty much always be faster than loading it from the data section. And we need to load it anyways. I can see global state useful for larger blocks of data, but a 4-byte array can just sit on the stack and is probably faster this way (if the compiler isn't optimizing it this way anyways).

```suggestion
  std::array<std::byte, 4> end_of_header{{
```

Reply:
Good point. I added the `constexpr` to  be sure we optimize out the constructor call at compile time. Then my mind short circuited to `static constexpr` without giving it much thought, but full ack on your statement.